packages/ember-glimmer/tests/integration/syntax/with-test.js

```diff
@@ -179,64 +186,45 @@ moduleFor('Syntax test: Multiple {{#with as}} helpers', class extends RenderingT
     this.assertText('Admin: Tom Dale User: Yehuda Katz');
   }
 
-  ['@test re-using the same variable with different #with blocks does not override each other']() {
```

Comment:
"re-using the same variable with different #with blocks does not override each other" is repeated twice